qermit/clifford_noise_characterisation/ccl.py

```diff
@@ -461,7 +491,7 @@ def gen_CDR_MitEx(
             MitEx(
                 device_backend,
                 _label="ExperimentMitex",
-                mitres=gen_compiled_MitRes(device_backend, 2),
+                mitres=gen_compiled_MitRes(device_backend, 0),
```

Comment:
Is it necessary to switch to optimisation level 0 given that the barriers have been added?

Reply:
Unfortunately this still seems to be a requirement for consistent results